<launch>
    <node pkg="sesame_navigation" type="drone_manager.py" name="drone_manager" output="screen">
        <param name="obstacle_radius" type="double" value="2.5"/>
        <param name="obstacle_epsilon" type="double" value="0.25"/>
        <param name="output_frame" type="string" value="map"/>
        <param name="pose_regex" type="string" value="^/uav_[1-9].*/local_position/pose_stamped$"/>
        <rosparam param="drones_id_map">
            {"uav_1":1 , "uav_2":2, "uav_3":3, "uav_4":4, "uav_5":5, "uav_6":6, "uav_7":7, "uav_8":8, "uav_9":9}
        </rosparam>
        <rosparam param="drones_to_track">
            ["uav_1", "uav_2", "uav_3", "uav_4", "uav_5", "uav_6", "uav_7", "uav_8", "uav_9"]
        </rosparam>
        <param name="use_visualization" type="bool" value="true"/> 
        <param name="marker_size" type="double" value="0.25"/>

        <remap from="input_drones" to="/vision_stack/drones"/>
        <remap from="output_drones" to="/vision_stack/filtered_drones"/>
        <remap from="visualization" to="/vision_stack/drones_visualization"/>
    </node>
</launch>

